base/setup/usetup/bootsup.c

```diff
@@ -88,6 +88,7 @@ typedef struct _BTRFS_BOOTSECTOR
     UCHAR Fill[1521]; // 1536 - 15
```

Comment:
This commit may be merged together with the other one "[USETUP][SETUPLIB] BTRFS boot sector (VBR) now installs correctly. "